engine/resources/net.go

```diff
@@ -119,9 +119,6 @@ func (obj *NetRes) Validate() error {
 	}
 
 	// validate network address input
-	if (obj.Addrs == nil) != (obj.Gateway == "") {
```

Comment:
Is this no longer true?

Reply:
I tried to explain it in the commit message, but there are scenarios where you'd want to set one without the other. For example, if you're setting up a secondary interface on a different subnet, you might not want to change the default gateway, as having 2 defaults (on the same routing table) leads to issues. That's why travis was losing track of the build when the shell tests added a gateway.